spec/helpers/brexit_checker_helper_spec.rb

```diff
@@ -57,4 +57,71 @@
       expect(subject).to contain_exactly('A', 'B')
     end
   end
+
+  describe "#next_question" do
+    let(:q1) { FactoryBot.build(:brexit_checker_question) }
+    let(:q2) { FactoryBot.build(:brexit_checker_question, criteria: [{ "all_of" => %w(a b) }]) }
```

Comment:
Minor: do we need to specify the criteria here any more?

Reply:
We need for the premise of the example on line 103 - i.e. a question has criteria, but it doesn't match so we skip it.

Reply:
I'm not sure what you mean here. Since my suggestion around the wording of the contexts is quite lengthy, perhaps we it would be easier to discuss in person?